.circleci/config.yml

```diff
@@ -68,6 +94,18 @@ commands:
       - run: make fakepackage
       - run: make e2e
 
+  run_tests_nightly:
+    steps:
+      - run: test -z `go fmt ./...`
+      - run: make lint
+      - run: make check
+      - run: make integration
+      - run: make test
+      - run: make test-generate
+      - run: make fakepackage
+      - run: make e2e
```

Comment:
copy pasta from `run_tests` but adding another step `make indexer-v-algod-latest`

Reply:
@algojack is there a recommended way to organize this sort of thing in circleci?

Reply:
we can do it with a pipe to combine all in one step, but every step gets it's own section in circleci, which is easier to troubleshoot (because we don't have to scroll through as much), so separate steps are good too. Here's an example for both: https://circleci.com/docs/2.0/configuration-reference/#shorthand-syntax